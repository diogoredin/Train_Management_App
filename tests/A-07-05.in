6
2
0
2
1
0
1
5
1
0
6
2
1
0
5
3
qqqqqq
1
0
0
