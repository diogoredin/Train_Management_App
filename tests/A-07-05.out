Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: == Passageiro 0: Obi-Wan ==

Itinerário 1 para 2017-10-18 @ 51.50
Serviço #180 @ 51.50
05:47 Porto - Campanhã
08:10 Casa Branca
11:23 Faro

Itinerário 2 para 2017-10-18 @ 32.76
Serviço #180 @ 21.92
05:47 Porto - Campanhã
08:10 Casa Branca
Serviço #694 @ 10.84
17:08 Casa Branca
18:36 Lisboa - Oriente
Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: == Passageiro 1: Yoda ==

Itinerário 1 para 2017-10-18 @ 21.92
Serviço #180 @ 21.92
05:47 Porto - Campanhã
08:10 Casa Branca
Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Identificador do passageiro: Mostrar itinerários associados a um passageiro: Operação inválida: O passageiro com o identificador 1 não existe.
Menu de gestão de itinerários
1 - Mostrar todos os itinerários
2 - Mostrar itinerários associados a um passageiro
3 - Registar itinerário para um passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: Nome do passageiro: Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: 0|qqqqqq|NORMAL|0|0.00|00:00
Menu de gestão de passageiros
1 - Mostrar todos os passageiros
2 - Mostrar passageiro
3 - Registar passageiro
4 - Mudar nome de passageiro
0 - Sair
Escolha uma opção: Menu principal
1 - Reiniciar
2 - Abrir
3 - Guardar
4 - Consulta de serviços
5 - Gestão de passageiros
6 - Gestão de itinerários
0 - Sair
Escolha uma opção: 